manual/mingw/tools/chocolateyInstall.ps1

```diff
@@ -1,64 +1,13 @@
-﻿# Default values
-$packageName = 'mingw'
-$packageVersion = '4.8.3'
-$rev = 'rev0'
-$threads = 'posix'
-$exception = 'default' #dwarf is 32bit only, seh is 64bit only, sjlj works with 32 / 64
+﻿. (Join-Path (Split-Path -parent $MyInvocation.MyCommand.Definition) chocolateyInclude.ps1)

```

Comment:
Does dot sourcing it make it run so that $zipFile is available?

I would prefer that there is no ambiguity and you use & to ensure it runs.